### ISSUE TITLE
Feature Request: Scan barcodes in Mobile browsers

### ISSUE THREAD

**Issue by davoezzz** [snipe/snipe-it#7036]
#### Expected Behavior (or desired behavior if a feature request)

The possibility to scan barcodes for example asset search or to fill in a serial number. Should be great that you can use some barcode scanner for this.

Example of some mobile barcode reader: https://demo.dynamsoft.com/dbr_wasm/barcode_reader_javascript.html

-----

#### Actual Behavior

It is not available at the moment.

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? - Fresh
- Version of Snipe-IT you're running - latest 4.5
- Version of PHP you're running - 7.0.3
- Version of MySQL/MariaDB you're running - 10.0
- What OS and web server you're running Snipe-IT on - Server 2016

**Comment by misilot** [comment]
Could something like the HTML5 File API be used to accomplish this on some of the fields such as Look up Asset Tag, Serial Number, etc...

**Comment by Cabe** [comment]
I would like this feature also.

It's worth nothing (I work with WebAR applications) that to access hardware (i.e. the camera) the URL needs to be secure.

**Comment by snipe** [comment]
> It's worth nothing (I work with WebAR applications) that to access hardware (i.e. the camera) the URL needs to be secure.

The url would be up to your installation, as always.

**Comment by misilot** [comment]
Yes it is still relevant.

**Comment by misilot** [comment]
Yes it is still relevant.

**Comment by misilot** [comment]
yes still relevant.

**Comment by aleksandrmetik** [comment]
+1

**Comment by misilot** [comment]
yes still relevant.

**Comment by misilot** [comment]
yes

**Comment by davoezzz** [comment]
Yes still relevant!

**Comment by Cabe** [comment]
Also still interested!

**Comment by tareko** [comment]
Agreed that this feature would be really helpful. Is there some way to contribute to it? I would be happy to contribute to a bounty for this.

tarek : )

**Comment by davoezzz** [comment]
Yes still relevant!

**Comment by misilot** [comment]
Yes

**Comment by BitTourette** [comment]
Hey there ... 
i think i could implement it. But i'm new to snipeit and not sure where to put all the stuff (buttons and options etc). I can try something in the serialnumer asset field with a "scan" button next to it an an special "scanner" option under to turn on and off the ability in the UI. Note sure if there will be soon a PR and whether it would be accepted.

**Comment by hesske** [comment]
+1 for this

**Comment by diegogarciadev** [comment]
Maybe you're interested in this application:

 Snipe-IT Assets Management - Scanner App
https://play.google.com/store/apps/details?id=com.diegogarciadev.assetsmanager.snipeit

Regards!

<blockquote><img src="https://play-lh.googleusercontent.com/qzzgYh2eJTHRYoCyp9AEy4bAaFQGL3XjuKhVCOj1yds5T7y7HT6YCm3oQU-vu-4dIfA" width="48" align="right"><div><strong><a href="https://play.google.com/store/apps/details?id=com.diegogarciadev.assetsmanager.snipeit&hl=en_US&gl=US">Snipe-IT Assets Management - Scanner App - Apps on Google Play</a></strong></div><div>Scan your Asset Tags and make basic operations with Snipe-IT API</div></blockquote>

**Comment by davoezzz** [comment]
@diegogarciadev i am, but we are working with Apple devices only ;(

**Comment by diegogarciadev** [comment]
> @diegogarciadev i am, but we are working with Apple devices only ;(

Ops... I can't help you then, regards

**Comment by derrickchiang** [comment]
+1 for this
